BE/app/core/config.py

```diff
@@ -0,0 +1,17 @@
+import os
+from pydantic_settings import BaseSettings, SettingsConfigDict
+from dotenv import load_dotenv
+load_dotenv()
+
+class Settings(BaseSettings):
+    GEMINI_API_KEY: str
+    NOTION_API_KEY: str
+
+    model_config = SettingsConfigDict(
+        case_sensitive=True,
+        env_file='.env',
+        env_file_encoding='utf-8'
+    )
+    
+
+settings = Settings()
```

Comment:
`Settings` 에 지정한 설정값이 `.env`에 존재하지 않는다면 해당 라인에서 에러가 발생하는데요.
예외처리 해주시면 좋겠습니다.

```suggestion
try:
    settings = Settings()
except Exception as e:
    raise RuntimeError(f"Fail to load environment vars: {e}")
```

Reply:
- settings 인스턴스 생성 시 예외 처리 추가: 4dc4a00960dc23ef6fc8ebcdbdeeceff9df35d4a